packages/itwin/ec3-widget/src/components/LabelAction.scss

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import '~@itwin/itwinui-css/scss/variables';
+
+.ec3w-label-details-container {
+  overflow-x: auto;
+  height: 100%;
+
+  .ec3w-label-details {
+    display: flex;
+    flex-direction: column;
+    gap: $iui-baseline;
+    margin-bottom: $iui-baseline;
+  }
+
+  .ec3w-label-field-legend {
+    color: $iui-color-system-gray;
+  }
+}
+
+.ec3w-pair-list {
+  flex-direction: column;
```

Comment:
I don't think this is doing anything without `display:flex`

Reply:
Deleted the ```flex-direction```